app/src/main/java/org/mozilla/fenix/perf/Stat.kt

```diff
@@ -64,6 +64,11 @@ open class Stat {
         return getStatText(pid).split(' ')[FIELD_POS_STARTTIME].toLong()
     }
 
+    fun getProcessStartTimeStampNano(pid: Int): Long {
+        val timeInTicks = getStatText(pid).split(' ')[FIELD_POS_STARTTIME].toLong()
```

Comment:
nit: This is duplication: can we call `getProcessStartTimeTicks()` and convert the result?

Reply:
thanks! fixed it